specification/trafficmanager/resource-manager/Microsoft.Network/2017-09-01-preview/trafficmanageranalytics.json

```diff
@@ -126,14 +126,11 @@
         ],
         "responses": {
           "200": {
-            "description": "The subscription-level Traffic Manager Realtime User Metrics key.",
+            "description": "The subscription-level Traffic Manager Real User Metrics key.",
             "schema": {
               "$ref": "#/definitions/TrafficManagerUserMetricsKeyModel" 
             }
           },
-          "204": {
```

Comment:
Does this mean that 204 is impossible, or you want now a 204 to raise an exception?

Reply:
The "Get" operation never returns a 204. If the subscription has no RUM key, it returns a 200 and the body returns a blank key ("key": "")